boards/arduino-mkr1000/Makefile.include

```diff
@@ -0,0 +1,35 @@
+# define the cpu used by Arduino/Genuino MKR1000 board
+export CPU = samd21
+
+ifeq ($(PROGRAMMER),jlink)
+    # in case J-Link is attached to SWD pins, use a plain CPU memory model
+    export CPU_MODEL = samd21g18a
+
+    # setup JLink for flashing
+    export JLINK_DEVICE := atsamw25
+    include $(RIOTMAKE)/tools/jlink.inc.mk
+
+    # set default serial port depending on operating system
+    PORT_LINUX ?= /dev/ttyUSB0
+    PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+else
+    # when BOSSA is used, use a different flash map
+    # refer https://github.com/shumatech/BOSSA for this programmer
+    export PROGRAMMER = bossa
+    export CPU_MODEL = samd21g18a_mkr1000
```

Comment:
Why a different CPU model here? If it is because of the different linkerscript, then it would be the cleaner solution to override the linkerscript variable here directly!